### PR TITLE
update date presentation and move getDate to utils

### PR DESCRIPTION
**Pull request by wouter1975**

### CODE REVIEW

File: packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx [hunk 1]
```diff
@@ -90,7 +92,9 @@ export const WfoStartPage = () => {
         listItems:
             processesSummaryResult?.processes.page.map((workflow) => ({
                 title: workflow.workflowName,
-                value: workflow.startedAt,
+                value: parseDateToLocaleDateTimeString(
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx:95]: Just checking: have you tried `formatDate(...)` -- I think that one does getDate and parseDate together.
Can be found in `date.ts`

Comment (wouter1975) [reply packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx:95]: I tried but not good enough, I throught it did not support the string types, but I retried and works fine. I will update the code.

File: packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx [hunk 2]
```diff
@@ -68,10 +68,10 @@ export const WfoStartPage = () => {
         headerStatus: SummaryCardStatus.Neutral,
         listTitle: t('activeSubscriptions.listTitle'),
         listItems:
-            subscriptionsSummaryResult?.subscriptions.page.map(
+            subscriptionsSummaryResult?.subscriptions?.page.map(
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx:71]: As far as i know is `subscriptions` not "possibly undefined". Same goes for the other cases in this PR.

**Review by DutchBen — APPROVED** [review]

File: packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx [hunk 3]
```diff
@@ -71,7 +71,7 @@ export const WfoStartPage = () => {
             subscriptionsSummaryResult?.subscriptions.page.map(
                 (subscription) => ({
                     title: subscription.description,
-                    value: getFirstUuidPart(subscription.subscriptionId),
+                    value: formatDate(subscription?.startDate),
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx:74]: The possibly undefined comment applies here too. The `subscription`, `workflow` and `task` cannot be undefined

**Review by ricardovdheijden — APPROVED** [review]

File: packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx (hunk 3 above)
Comment (wouter1975) [reply packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx:74]: ok

**Pull request merged**